# Justin Guiel
My educational and research backgrounds are strictly biology with heavy focus on molecular genetics and microbiology. I have a data, cloud, and software engineer professionally since early 2021. I have been consistently teaching myself compsci over the past 3+ years through textbooks, projects, and resources like MIT's OCW. My desire to learn more has only grown stronger.

## Currently
- Contributer for Transcriptic's Autoprotocol library as engineer
- Scientific Computing SWE developing lab automation workflows for small molecule/drug discovery

## Previously
- Precision medicine data infrastructure and mgmt engineer
- Post-analysis data processing pipelines, production bioinformatics ops
- Cloud infrastructure (AWS)
- ETL pipelines (AWS)
- Linux server data management, migrate on-prem to cloud
- NGS Illumina Sequencing, whole-exome/transcriptome on NovaSeq
- MALDI-TOF Sequencing for SNP analysis
- Teacher: IB & GCSE Biology, USHS chemistry, USHS math

*Reach me at jfguiel@gmail.com*

## I Speak
Python, AWS, SQL, Bash/shell, DjangoORM, genetics, server mgmt, Linux/Unix env, data transformation, NoSQL, lab automation, and front-end

## Currently learning
- JavaScript
- Apache Airflow
- Rust

## Currently Reading
- Server Load Balancing - Tony Bourke, O'Reilly
- The Rust Programming Language - Steve Klabnik and Carol Nichols
